modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc

```diff
@@ -149,6 +149,13 @@ The IPv6 address can be any address as long as it is not routable via the `barem
 Ensure that UEFI is enabled and UEFI PXE settings are set to the IPv6 protocol when using IPv6 addressing.
 ====
 
+. Modify the provisioner network.
```

Comment:
This command sets manual IPv4 address on provisioning.
Not clear here if this is always needed. In the previous it shows setting ipv6.

Reply:
So would this change not be needed?